### PR TITLE
Adding info on some ios14 beta issues

### PR DESCRIPTION
**Pull request by sfshaza2**
Staged:  https://sz-flutter.firebaseapp.com/docs/development/ios-14

This is probably overkill on the number of reviewers, but speak up if you feel strongly! I'd like to land this tonight.

### CODE REVIEW

File: src/docs/development/ios-14.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+title: Developing for iOS 14 beta
+description: Describes a couple gotchas (and how to work around them)when developing for iOS14 on Flutter 1.20.
```
Comment (jmagman) [inline src/docs/development/ios-14.md:3]: Missing space between `)w`?

File: src/docs/development/ios-14.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+## Launching the app from the home screen
+
+When you install a debug version of your app on a physical
+iOS 14 device (using `flutter run`), then disconnect,
+and then try to open the app from the home screen, it crashes.
+
+**Solution**: Use `flutter run --release` (or `--profile`)
+if you want to start your app from the home screen.
+
+Once again, the crash only happens in debug mode,
```
Comment (jmagman) [inline src/docs/development/ios-14.md:41]: Can we mention that it doesn't happen in the simulator?

File: src/docs/development/ios-14.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+Only the debug version of your app needs to access the local network.
+
+For more information,
+see [Issue 60634]({{site.github}}/flutter/flutter/issues/60634).
+
+
+## Launching the app from the home screen
+
+When you install a debug version of your app on a physical
+iOS 14 device (using `flutter run`), then disconnect,
```
Comment (jmagman) [inline src/docs/development/ios-14.md:35]: (using `flutter run` or Xcode)

File: src/docs/development/ios-14.md (hunk 2 above)
Comment (sfshaza2) [reply src/docs/development/ios-14.md:41]: Done

File: src/docs/development/ios-14.md (hunk 1 above)
Comment (sfshaza2) [reply src/docs/development/ios-14.md:3]: done

File: src/docs/development/ios-14.md (hunk 3 above)
Comment (sfshaza2) [reply src/docs/development/ios-14.md:35]: done

**Comment by sfshaza2** [comment]
Thanks for your quick review, @jmagman!

**Pull request merged**